src/uart.rs

```diff
@@ -1899,15 +1899,15 @@ fn new_common<UART: Uart>(
         uart_driver_install(
             UART::port(),
             if rx.is_some() {
-                config.rx_fifo_size as _
+                config.rx_fifo_size
             } else {
-                0
-            },
+                UART_FIFO_SIZE * 2
```

Comment:
Not an issue: I do agree with you, the `assert` in the ESP IDF C source code needs this (though it is still weird, especially if you look at the line below (how TX is handled)). 

Just wondering: why `* 2` and not - say - `+ 1`?

Reply:
I just take [default](https://github.com/esp-rs/esp-idf-hal/blob/1faefe4dc2d70ada92ddf1e2bf9418bcfd194aa4/src/uart.rs#L496) value